rclcpp/include/rclcpp/utilities.hpp

```diff
@@ -53,7 +53,11 @@ namespace rclcpp
  */
 RCLCPP_PUBLIC
 void
-init(int argc, char const * const argv[], const InitOptions & init_options = InitOptions());
+init(
+  int argc,
+  char const * const argv[],
+  const InitOptions & init_options = InitOptions(),
+  bool install_sigterm_handler = true);
```

Comment:
Also, I wonder if we should even install a handler if the corresponding `shutdown_on_<signal>`  flag is false.

Reply:
> @ivanpauno meta: shall we make this an init option (and put it in the InitOptions struct)?

`Context.init()` doesn't initialize signal handlers, while `rclcpp::init()` does.
Thus, I added an extra argument to init.

Reply:
Good point.

Reply:
This feels weird to me... what if someone wants to install the sigterm but not the sigint handler?

It seems like we need another configuration instead, which has a bool for each, and then it feels more natural to see "init options" and "signal handler options" as the two arguments to `init()` since it is "context.init" + "install_signal_handlers".

Reply:
Adding a "signal handler options" sounds good to me, it could be an enum "install sigint/install sigterm/install both/ install none"

Reply:
Done, see https://github.com/ros2/rclcpp/pull/1771/commits/b4d87ea24b63c2abfe412b0240dcfa8ed1c7ee69.